2024-04-16 08:54:53,726 INFO    MainThread:22232 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-16 08:54:53,726 INFO    MainThread:22232 [wandb_setup.py:_flush():76] Configure stats pid to 22232
2024-04-16 08:54:53,726 INFO    MainThread:22232 [wandb_setup.py:_flush():76] Loading settings from C:\Users\vomin\.config\wandb\settings
2024-04-16 08:54:53,726 INFO    MainThread:22232 [wandb_setup.py:_flush():76] Loading settings from d:\40-Day Python Practice\wandb\settings
2024-04-16 08:54:53,726 INFO    MainThread:22232 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-16 08:54:53,727 INFO    MainThread:22232 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-04-16 08:54:53,727 INFO    MainThread:22232 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-16 08:54:53,727 INFO    MainThread:22232 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-16 08:54:53,727 INFO    MainThread:22232 [wandb_init.py:_log_setup():521] Logging user logs to d:\40-Day Python Practice\wandb\run-20240416_085453-8n8q2g2j\logs\debug.log
2024-04-16 08:54:53,727 INFO    MainThread:22232 [wandb_init.py:_log_setup():522] Logging internal logs to d:\40-Day Python Practice\wandb\run-20240416_085453-8n8q2g2j\logs\debug-internal.log
2024-04-16 08:54:53,727 INFO    MainThread:22232 [wandb_init.py:_jupyter_setup():467] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001C4C0CEA0C0>
2024-04-16 08:54:53,727 INFO    MainThread:22232 [wandb_init.py:init():561] calling init triggers
2024-04-16 08:54:53,727 INFO    MainThread:22232 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.01, 'epochs': 1000}
2024-04-16 08:54:53,727 INFO    MainThread:22232 [wandb_init.py:init():611] starting backend
2024-04-16 08:54:53,727 INFO    MainThread:22232 [wandb_init.py:init():615] setting up manager
2024-04-16 08:54:53,729 INFO    MainThread:22232 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-04-16 08:54:53,729 INFO    MainThread:22232 [wandb_init.py:init():623] backend started and connected
2024-04-16 08:54:53,739 INFO    MainThread:22232 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-04-16 08:54:53,739 INFO    MainThread:22232 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-04-16 08:54:53,739 INFO    MainThread:22232 [wandb_init.py:init():715] updated telemetry
2024-04-16 08:54:53,779 INFO    MainThread:22232 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-04-16 08:54:54,665 INFO    MainThread:22232 [wandb_run.py:_on_init():2357] communicating current version
2024-04-16 08:54:55,373 INFO    MainThread:22232 [wandb_run.py:_on_init():2366] got version response 
2024-04-16 08:54:55,373 INFO    MainThread:22232 [wandb_init.py:init():799] starting run threads in backend
2024-04-16 08:54:55,768 INFO    MainThread:22232 [wandb_run.py:_console_start():2335] atexit reg
2024-04-16 08:54:55,768 INFO    MainThread:22232 [wandb_run.py:_redirect():2190] redirect: wrap_raw
2024-04-16 08:54:55,768 INFO    MainThread:22232 [wandb_run.py:_redirect():2255] Wrapping output streams.
2024-04-16 08:54:55,768 INFO    MainThread:22232 [wandb_run.py:_redirect():2280] Redirects installed.
2024-04-16 08:54:55,769 INFO    MainThread:22232 [wandb_init.py:init():842] run started, returning control to user process
2024-04-16 08:56:46,210 INFO    MainThread:22232 [wandb_run.py:_finish():2064] finishing run minhbao1705/advesting_loss/8n8q2g2j
2024-04-16 08:56:46,211 ERROR   MainThread:22232 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-04-16 08:56:46,211 INFO    MainThread:22232 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-16 08:56:46,211 INFO    MainThread:22232 [wandb_init.py:_jupyter_teardown():449] cleaning up jupyter logic
2024-04-16 08:56:46,211 INFO    MainThread:22232 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-04-16 08:56:46,212 INFO    MainThread:22232 [wandb_run.py:_restore():2287] restore
2024-04-16 08:56:46,212 INFO    MainThread:22232 [wandb_run.py:_restore():2293] restore done
